client/src/Profile.js

```diff
@@ -0,0 +1,223 @@
+import React from 'react';
+import './Profile.css';
+import Tag from './TagCopy.js';
+import cofedlogo from './assets/CoFEDlogo.png';
+import instagram from './assets/instagram.svg';
+import facebook from './assets/facebook.svg';
+import tina from './assets/tina.png';
+import plusSign from './assets/plus-sign.svg';
+
+export default function Profile(props) {
+  const [edit, setEdit] = React.useState(false);
+  const [name, setName] = React.useState(props.name); //from props
+  const [location, setLocation] = React.useState(props.location); //from props
+
+  const [profilePic, setProfilePic] = React.useState(props.profilePic);
+  const [phone, setPhone] = React.useState(props.phone);
+  const [tags, setTags] = React.useState(props.tags);
+  const [mission, setMission] = React.useState(props.missionText);
+  const [description, setDescription] = React.useState(props.descText);
+  const [instaLink, setInstaLink] = React.useState(props.instaLink);
+  const [fbLink, setFbLink] = React.useState(props.fbLink);
+
+  const [website, setWebsite] = React.useState(props.website);
+  const [websiteLink, setWebsiteLink] = React.useState(props.websiteLink);
+  const [email, setEmail] = React.useState(props.email);
+  const [emailLink, setEmailLink] = React.useState(props.emailLink);
+
+  function handleDelete(tagIndex) {
+    const newTags = tags.filter((tag, i) => i != tagIndex);
+    setTags(newTags);
+  }
+
+  function toggleEdit() {
+    //edit button function
+    setEdit(!edit);
+  }
+
+  function renderEdit() {
+    return (
+      <div className="blueBorder">
+        <div className="topDiv">
+          <div className="logoDiv">
+            <a href="https://www.cofed.coop/">
+              <img src={cofedlogo} className="logo" />
```

Comment:
we can take out the logo because we have the logo in the top left corner of the app already and it'll be redundant